turbinia/processors/mount_local.py

```diff
@@ -482,12 +515,21 @@ def PostprocessDeleteLosetup(device_path, lv_uuid=None):
 
     # Check that the device was actually removed
     losetup_cmd = ['sudo', 'losetup', '-a']
-    log.info('Running: {0:s}'.format(' '.join(losetup_cmd)))
-    try:
-      output = subprocess.check_output(losetup_cmd)
-    except subprocess.CalledProcessError as e:
-      raise TurbiniaException(
-          'Could not check losetup device status {0!s}'.format(e))
+    for _ in range(RETRY_MAX):
+      log.info('Running: {0:s}'.format(' '.join(losetup_cmd)))
+      try:
+        output = subprocess.check_output(losetup_cmd)
+      except subprocess.CalledProcessError as e:
+        raise TurbiniaException(
+            'Could not check losetup device status {0!s}'.format(e))
+      if output.find(device_path.encode('utf-8')) != -1:
+        log.info(
+            'Loop device {0:s} mode is {1}'.format(
+                device_path,
+                os.stat(device_path).st_mode))
+      else:
+        break
+    # Fail if loop device was not removed.
```

Comment:
I think the line below this one has the same issue with the `find` line above (could give a FP for `/dev/loop1` when `/dev/loop10` exists).

Reply:
Please see above, I don't think that may be the case per debugger I placed to test the condition